.github/workflows/build.yml

```diff
@@ -34,8 +34,11 @@ jobs:
       - name: "Install python dependencies"
         run: |
           echo ::group::PYDEPS
-          pip install packaging twine wheel mkdocs jq
+          pip install packaging twine wheel mkdocs jq PyGObject<3.45
```

Comment:
```suggestion
          pip install packaging twine wheel mkdocs jq "PyGObject<3.45"
```

bash is seeing the '<' in ways you do not want :)

Reply:
:man_facepalming:

Reply:
I don't see why it should be needed here.